### DISCUSSION TITLE
v1.0-beta

### DISCUSSION THREAD

**Discussion by dev-brendanprice** [Asfalto-Ascari-Group/EchelonClient-Release-Stable#2] [General]
Project Echelon beta release, Partially working but not 100% up to standard.
**_This version will mainly serve as an update server for any clients past this version._**

Please leave any issues you find in discussions/General

<hr /><em>This discussion was created from the release <a href='https://github.com/Asfalto-Ascari-Group/EchelonClient-Release-Stable/releases/tag/BetaRelease'>v1.0-beta</a>.</em>

**Comment by EXYZED** [comment]
Thank you for creating this release.
